services/slam/builtin/builtin_test.go

```diff
@@ -1117,6 +1117,10 @@ func TestGetMapAndPosition(t *testing.T) {
 	test.That(t, pc, test.ShouldBeNil)
 	test.That(t, fmt.Sprint(err), test.ShouldContainSubstring, "error getting SLAM map")
 
+	internalState, err := svc.GetInternalState(context.Background(), "hi")
```

Comment:
Note: most testing of endpoints has to be done in integration tests. This just confirms the failure/error statement